src/core_modules/capture-core/components/WidgetEventSchedule/WidgetEventSchedule.container.js

```diff
@@ -41,28 +35,35 @@ export const WidgetEventSchedule = ({
     const [scheduleDate, setScheduleDate] = useState(suggestedScheduleDate);
     const { events } = useEventsInOrgUnit(orgUnitId, scheduleDate);
 
-    const [mutate] = useDataMutation(scheduleEventMutation, {
-        onComplete: () => {
-            history.push(`/enrollment?${urlArguments({ orgUnitId, programId, stageId, enrollmentId })}`);
-        },
-    });
     const eventCountInOrgUnit = events
         .filter(event => moment(event.dueDate).format('YYYY-MM-DD') === scheduleDate).length;
 
-    const onHandleSchedule = async () => {
-        if (!scheduleDate) { return; }
-        await mutate({ events: [{
-            dueDate: scheduleDate,
-            dataValues: [],
-            trackedEntityInstance: teiId,
-            orgUnit: orgUnitId,
-            enrollment: enrollmentId,
-            program: programId,
-            programStage: stageId,
-            status: 'SCHEDULE',
-            notes: [],
-        }] });
-    };
+    useEffect(() => {
+        dispatch(startEditingScheduleEvent());
+    }, []); // eslint-disable-line
```

Comment:
Hey,
The best practice for `useEffect` is to either pass an array with all the dependencies or pass nothing. You can read more about it [here](https://github.com/facebook/create-react-app/issues/6880#issuecomment-485912528). Therefore, I am curious to know why did you disable the `lint` checks in this case? 
Thanks!

Reply:
hey because I only want to run it once on mount. Thanks